testonly/hasher.go

```diff
@@ -17,7 +17,16 @@ package testonly
 // This file implements the hashing functions that are part of a Trillian
 // personality.
 
-import "crypto/sha256"
+import (
+	"crypto"
+	"crypto/sha256"
+
+	"github.com/google/trillian/merkle/rfc6962"
+)
+
+// Hasher is the default hasher for tests.
+// TODO: Make this a custom algorithm to decouple hashing from coded defaults.
```

Comment:
Can we do this soon after this PR? It would show up if there's any RFC6962 specific assumptions in the generic code. There shouldn't be but it's possible.